tools/PI/DevHome.PI/Controls/AddToolControl.xaml.cs

```diff
@@ -101,55 +161,290 @@ private void ClearValues()
             return null;
         }
 
-        var argType = ExternalToolArgType.None;
-
-        if (HwndRadio.IsChecked ?? false)
+        var activationType = ToolActivationType.Launch;
+        if (ProtocolRadio.IsChecked ?? false)
         {
-            argType = ExternalToolArgType.Hwnd;
+            activationType = ToolActivationType.Protocol;
         }
-        else if (ProcessIdRadio.IsChecked ?? false)
+        else if (selectedApp is not null && selectedApp.IsMsix)
```

Comment:
I believe this can be simplified to `selectedApp?.IsMsix`

Reply:
No, tried that, doesn't work.